gulpfile.js

```diff
@@ -80,7 +80,9 @@ gulp.task('copy', ['clean'], function() {
       './node_modules/handlebars/dist/handlebars.min.js',
       './node_modules/underscore/underscore-min.js',
       './node_modules/backbone/backbone-min.js',
+      // './lib/highlight.9.9.0.pack.js',
       './lib/highlight.7.3.pack.js',
```

Comment:
No, just keeping with the current version until we can totally delete these vendor files and use proper dependency management instead. I just opened a new issue for that: #12